src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -105,7 +115,7 @@ protected boolean isFinished() {
             }
 
             @Override
-            protected void end() {
+            protected void end() {  
```

Comment:
... There's a space there.

... Why is there a space there?

Reply:
control shift f is king